app/scenes/account_notifications/account_notifications.js

```diff
@@ -177,7 +170,6 @@ export default class AccountNotifications extends PureComponent {
     }
 
     saveUserNotifyProps = () => {
-        EventEmitter.emit(SAVING_NOTIFY_PROPS, true);
```

Comment:
Ended up having to remove this. Couldn't get the save request to complete with it added. Looked like it was causing the request to be fired twice.